httphandler/storage/apiserver_test.go

```diff
@@ -408,3 +412,100 @@ func TestGetManifestObjectLabelsAndAnnotations(t *testing.T) {
 		})
 	}
 }
+
+//go:embed testdata/role_1.json
+var roleJson1 []byte
+
+//go:embed testdata/rolebinding_1.json
+var roleBindingJson1 []byte
+
+//go:embed testdata/role_2.json
+var roleJson2 []byte
+
+//go:embed testdata/rolebinding_2.json
+var roleBindingJson2 []byte
+
+//go:embed testdata/role_3.json
+var roleJson3 []byte
+
+//go:embed testdata/rolebinding_3.json
+var roleBindingJson3 []byte
```

Comment:
issue: embedding contents from `testdata` files spawns unnecessary variables really fast

If we’re using `testdata`, let’s use it the way it was intended to be used: to load files located in the `testdata` directory.

Reply:
fixed